Reed/Reader/views/ReaderView.swift

```diff
@@ -12,52 +12,44 @@ import SwiftUIPager
 
 struct ReaderView: View {
     @ObservedObject var viewModel: ReaderViewModel
-    @ObservedObject var definitionModalViewModel: DefinitionModalViewModel = DefinitionModalViewModel()
+    @ObservedObject var definitionViewModel: DefinitionViewModel = DefinitionViewModel()
     @State var page: Int = 0
     @State private var currentPosition: CGSize = .zero
     @State private var newPosition: CGSize = .zero
+    @State private var bottomSheetShown = false
 
     init(ncode: String) {
         self.viewModel = ReaderViewModel(ncode: ncode)
     }
     
     var body: some View {
-        VStack(alignment: .center) {
-            Pager(
-                page: $page,
-                data: viewModel.items,
-                id: \.self,
-                content: { text in
-                    VStack(alignment: .center, spacing: /*@START_MENU_TOKEN@*/nil/*@END_MENU_TOKEN@*/) {
-                        TextView(text: text, defineSelection: definitionModalViewModel.generateContent)
-                    }
-            }).alignment(.start).preferredItemSize(CGSize(
-                width: UIScreen.main.bounds.width,
-                height: UIScreen.main.bounds.height * 0.6
-            ))
+        ZStack {
+            VStack(alignment: .center) {
+                Pager(
+                    page: $page,
+                    data: viewModel.items,
+                    id: \.self,
+                    content: { text in
+                        VStack(alignment: .center, spacing: /*@START_MENU_TOKEN@*/nil/*@END_MENU_TOKEN@*/) {
+                            TextView(text: text, defineSelection: definitionViewModel.generateContent)
+                        }
+                }).alignment(.start).preferredItemSize(CGSize(
+                    width: UIScreen.main.bounds.width,
+                    height: UIScreen.main.bounds.height * 0.6
+                ))
+                Text("\(page + 1)")
+                Spacer()
+                    .frame(height: UIScreen.main.bounds.height * 0.06)
+            }
+            GeometryReader { geometry in
```

Comment:
~~Does every `BottomSheetView` need a `GeometryReader`? If so, then would it be better to include the `GeometryReader` as a part of the `BottomSheetView` component?~~
Doesn't every `BottomSheetView` already have a `GeometryReader`? Can we just add `.edgesIgnoringSafeArea(.all)` to that instead, or will different views want to customize that differently?

Reply:
done

Reply:
we need to provide max height for the bottomsheet in readerview